### PR TITLE
add func calk tringle perrimetr

### PR DESCRIPTION
**Pull request by lislisgrom**
I'd like to halp you

### CODE REVIEW

**Review by lislisgrom — COMMENTED** [review]
Need to change logikal function

File: src/modules/perimetr.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-def perrimetr_func():
-    pass
\ No newline at end of file
+def perrimetr_func(a, b, c):
+    return a + b + c
```
Comment (zinarizinazina) [inline src/modules/perimetr.py]: Whats will be happend if i need to calc perrimetr athe figure ?

**Review by zinarizinazina — COMMENTED** [review]
This code need to chang logikal

File: src/modules/perimetr.py (hunk 1 above)
Comment (lislisgrom) [reply src/modules/perimetr.py]: i see your comments. i'll fix it.

**Pull request closed without merging**